src/context-menu/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import { createICacheMiddleware } from '@dojo/framework/core/middleware/icache';
+import theme from '../middleware/theme';
+import Menu, { MenuOption } from '../menu/index';
+import * as menuCss from '../theme/default/menu.m.css';
+import * as css from '../theme/default/context-menu.m.css';
+import * as fixedCss from './context-menu.m.css';
+
+export interface ContextMenuProperties {
+	options: MenuOption[];
+	onValue(value: string): void;
```

Comment:
As this is a menu with selection, do you think `onSelect` might be better?

Reply:
Sure, I'll update that.